app/services/invite_provider_user.rb

```diff
@@ -42,6 +43,14 @@ def send_welcome_email
     ProviderMailer.account_created(@provider_user).deliver_later
   end
 
+  def send_slack_notification
+    providers = @provider_user.providers.map(&:name).to_sentence
+    message = "#{@provider_user.first_name} has been invited to join #{providers}"
+    url = Rails.application.routes.url_helpers.provider_interface_provider_user_url(@provider_user)
```

Comment:
Considering this notification is for internal use, how about linking to a support interface url instead?

Reply:
:+1: Thanks @mnacos, amended here https://github.com/DFE-Digital/apply-for-teacher-training/pull/2822/commits/4cb64dfcf1d8c842849cbae2ceaa73bbfdce2727